docs/source/support/faq.rst

```diff
@@ -0,0 +1,183 @@
+==========================
+Frequently Asked Questions
+==========================
+
+What is the recommended specifications for running a nucypher?
+-------------------------------------------------------------------
+
+Worker nodes need to run ``nucypher`` and a local ethereum node, currently on the Goerli network. In total, you will
+require at 4GB for RAM. Nodes also need 24/7 uptime and a static, public IPv4 address.
```

Comment:
Will 4GB still be the requirement for a mainnet geth node?

Reply:
Only if it's light. The real, "full full" node would require, like, 16-32 GB...
Light sync fairly easily on mainnet though

Reply:
I could remove the sentence `In total, you will require at 4GB for RAM.` and leave the link to our specs. 

Perhaps that section of the docs should be updated? - https://docs.nucypher.com/en/latest/guides/installation_guide.html#system-requirements-and-dependencies

Reply:
I have been having issues with 4GB.  It is barely enough.  Sometimes I need to restart Beth to get it to let go of some RAM or I can’t get past the Scrypt.